guides/hosting/infrastructure/reverse-http-cache.md

```diff
@@ -426,6 +426,11 @@ sub vcl_backend_response {
 sub vcl_deliver {
     ## we don't want the client to cache
     set resp.http.Cache-Control = "max-age=0, private";
+    
+    #caching static assets is fine through
+    if (req.url ~ "\.(jpg|jpeg|png|gif|css|js|woff|woff2|svg|eot|webp)$") {
```

Comment:
We should implement the cache-control more smart like in Fastly. See https://github.com/shopware/recipes/blob/main/shopware/fastly-meta/6.4/config/fastly/deliver.vcl#L6 as 

Should be able to implement it like there

Reply:
what does that fastly code do?